ansible/roles/ocp4-workload-authentication/defaults/main.yml

```diff
@@ -32,3 +32,6 @@ ocp4_idm_htpasswd_user_count: 200
 
 # Remove Kubeadmin user upon successful installation of Authentication
 ocp4_idm_remove_kubeadmin: true
+
+# number of users in the htpasswd
+ocp4_idm_htpasswd_user_count: 200
```

Comment:
This already exists in line 21, doesn't it?

Reply:
yes! removing